nkms/blockchain/eth/chains.py

```diff
@@ -84,41 +86,28 @@ def wait_for_receipt(self, txhash, timeout=None) -> None:
         result = self.provider.w3.eth.waitForTransactionReceipt(txhash, timeout=timeout)
         return result
 
-    def time_travel(self, hours=None, seconds=None):
-        """Wait the specified number of wait_hours by comparing block timestamps."""
-        if hours:
-            duration = hours * 60 * 60
-        elif seconds:
-            duration = seconds
-        else:
-            raise Exception("Invalid time")
-
-        end_timestamp = self.provider.w3.eth.getBlock('latest').timestamp + duration
-        self.provider.w3.eth.web3.testing.timeTravel(end_timestamp)
-        self.provider.w3.eth.web3.testing.mine(1)
-
-    def spawn_miners(self, miner_agent, addresses: list, locktime: int, random_amount=False) -> list:
-
+    def time_travel(self, hours: int=None, seconds: int=None, periods: int=None):
         """
-        Deposit and lock a random amount of tokens in the miner escrow
-        from each address, "spawning" new Miners.
+        Wait the specified number of wait_hours by comparing
+        block timestamps and mines a single block.
         """
-        from nkms.blockchain.eth.actors import Miner
 
-        miners = list()
-        for address in addresses:
-            miner = Miner(miner_agent=miner_agent, address=address)
-            miners.append(miner)
+        querytime = list(filter(lambda t: bool(t), (hours, seconds, periods)))
```

Comment:
Ah, interesting way...
Can also do `if sum(map(bool, [hours, seconds, periods])) > 1:`

Reply:
Haha, yeah, that is a bit more readable.  I just needed a quick way to easily verify that only one arg is passed in.